src/desktop/apps/auction/queries/v2/me.ts

```diff
@@ -0,0 +1,50 @@
+import { graphql } from "lib/graphql"
+
+export const meV2Query = graphql`
```

Comment:
I tried to generate types using this trick, but is failing due to the following error:

```
$ yarn relay
...

ERROR:
Encountered 2 errors:
- Relay does not allow aliasing fields to `id`. This name is reserved for the globally unique `id` field on `Node`.

  Source: desktop/apps/auction/queries/v2/sale.ts (5:7)
  4:       _id: internalID
  5:       id: slug
           ^
  6:       associated_sale: associatedSale {
```

However, with this syntax the editor can apply syntax highlight and Prettier can also prettify the query, which is a big improvement.